modules/nf-core/samtools/fastq/tests/main.nf.test

```diff
@@ -28,11 +28,14 @@ nextflow_process {
         then {
             assertAll(
                 { assert process.success },
-                { assert snapshot(process.out.fastq[0][1].collect { path(it).linesGzip[0..6] }).match("bam_fastq") },
-                { assert snapshot(process.out.interleaved).match("bam_interleaved") },
-                { assert snapshot(file(process.out.singleton[0][1]).name).match("bam_singleton") },
-                { assert snapshot(file(process.out.other[0][1]).name).match("bam_other") },
-                { assert snapshot(process.out.versions).match("bam_versions") }
+                { assert snapshot(
+                    process.out.fastq[0][1].collect { path(it).linesGzip[0..6] },
+                    process.out.interleaved,
+                    file(process.out.singleton[0][1]).name,
+                    file(process.out.other[0][1]).name,
+                    process.out.versions
+                    ).match()
```

Comment:
nit: I am curious why this syntax is preferred?  While perhaps more concise, it now requires (1) me to go back and look at the `*.test.snap` to see what each item is matching, and (2) couples the _ordering_ in the `*.test.snap` with the _ordering_ here.  The latter to me is a footgun.  I'd prefer the explicit declaration.

Reply:
Hmm. I like it because it is more concise, and it doesn't need you to name each individual snapshot (nf-test allows you to leave the `match()` unnamed if you have only one per test context).

The ordering is fixed between the test and the snapshot, I never alter the snapshot manually, so not sure when that might be a problem.